android/README.md

```diff
@@ -0,0 +1,443 @@
+1. In the created mobile application in the Asgardeo, go to the advanced section and enable the client attestation.
+2. You need to subscribe to the "Google Play Integrity API" from your Google cloud project, you can do this from Enabled API's & Services in your Google cloud project.
+3. After that you need to create a new service account in the google project. You can create a Service Account for yourself with the following steps.
+
+    1. Go to `IAM & Admin` -> `Service Accounts`.
+    2. Click Create Service Account.
+    3. Fill the name and click create and continue.
+    4. You need to grant your service account the roles of Service Account User and Service Usage Consumer.
+    5. Click continue and then Done
+    6. You can see the service account added without keys, click : Actions -> Manage Keys for the service account.
+    7. Click Add key and Select JSON.
+    8. Save the JSON in secure place (We need this for Android Attestation Credentials for application metadata)
+
+4. After that, Update Application Advanced properties. The application you created requires 2 properties to perform android attestation.
+
+    1. Android package name
+    2. androidAttestationServiceCredentials
+       The JSON secret of Service Account downloaded. Note that this attribute is defined as a JSON object hence use the JSON key as it is.
+
+5. Now you can call the "Google Play Integrity API" from your application and pass the integrity token that you will get from the API result to the integrityToken value of the AuthenticationCoreConfig
+
+```kotlin
+private val asgardeoAuth: AsgardeoAuth = AsgardeoAuth.getInstance(
+    AuthenticationCoreConfig(
+        baseUrl = "https://localhost:9443",
+        redirectUri = "wso2sample://oauth2",
+        clientId = "<client_id>",
+        scope = "openid",
+        integrityToken = "<integrity_token>"
+    )
+)
+```
+
+## Use authenticators with the SDK
+
+The Asgardeo Auth SDK provides out-of-the-box support for some authenticators, which are accessible via the `AuthenticationProvider`. Each of the following functions will emit the aforementioned `AuthenticationStates`, except for the `AuthenticationState.Initial`.
+
+Before utilizing these authenticators, you need to integrate them into your application's login flow. You can find more information about this in the following link: [link_to_documentation].
+
+### Use any authentication mechanism
+
+If you are using any other authentication mechanism like email OTP, you can use the `authenticate` function. For this, you need to pass the authenticator id or authenticator which can be retrieved from the `authenticationFlow` returned from the `Authentication.Unauthenticated` state.
+
+This can be used in two ways:
+
+#### Authenticator parameters are known
+
+If you are aware of the authenticator parameters required for the authenticator (which can be found in the following link), you can directly call this function to authenticate the user with this authenticator.
```

Comment:
`which can be found in the following link`. There doesn't seem to be a link embedded here.

Reply:
this document is in development, hence removing this sentence.

Reply:
Fixed with https://github.com/asgardeo/mobile-ui-sdks/pull/1/commits/67ea36cc9fef864e59221f887574afe50546eea3